src/SFA.DAS.Payments.FundingSource.Application/SFA.DAS.Payments.FundingSource.Application.csproj

```diff
@@ -1,26 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="4.8.1" />
+    <PackageReference Include="Autofac" Version="8.0.0" />
     <PackageReference Include="AutoMapper" Version="7.0.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.31" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.0.5" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="NServiceBus" Version="8.1.1" />
+    <PackageReference Include="NServiceBus.Persistence.AzureStorage" Version="2.4.2" />
     <PackageReference Include="Polly" Version="7.1.0" />
     <PackageReference Include="SFA.DAS.Account.Api.Client" Version="1.6.1469" />
     <PackageReference Include="ESFA.DC.Logging.Interfaces" Version="0.1.4-beta0001" />
-    <PackageReference Include="SFA.DAS.Payments.Application" Version="1.1.0" />
-    <PackageReference Include="SFA.DAS.Payments.DataLocks.Messages" Version="1.0.0" />
-    <PackageReference Include="SFA.DAS.Payments.PeriodEnd.Messages" Version="1.0.0" />
-    <PackageReference Include="SFA.DAS.Payments.RequiredPayments.Messages" Version="1.0.0" />
-    <PackageReference Include="SFA.DAS.Payments.ServiceFabric.Core" Version="1.1.0" />
+    <PackageReference Include="SFA.DAS.Payments.Application" Version="2.0.0-prerelease-24" />
+    <PackageReference Include="SFA.DAS.Payments.DataLocks.Messages" Version="1.0.1" />
```

Comment:
DataLocks and RequiredPayments have new prerelease version for .NET 6. Not a massive thing, but worth updating to see if anything breaks. Plus any other DAS packages probably should be updated.